src/pages/metapackages/open-source/index.md

```diff
@@ -50,7 +50,15 @@ At this point, you should see symlinks for all the `pwa` modules inside the `ven
 -  Run a Magento installation with additional modules.
 -  Develop locally using the standard git workflow.
 
-You may need to ensure that there are no `Magento_PWA*` modules listed as `enabled` when you run `bin/magento module:status`. If there are, [follow the docs on how to enable modules](https://developer.adobe.com/commerce/php/development/build/component-management/).
+You may need to ensure that there are no `Magento_*Pwa` modules listed as `enabled` when you run `bin/magento module:status`. If there are, [follow the docs on how to enable modules](https://developer.adobe.com/commerce/php/development/build/component-management/).
```

Comment:
I know you didn't change this line, but the guidance to ensure that no modules are "_enabled_" seems odd... Is it possible that this is a mistake and should say "ensure no modules are _disabled_...". that would make more sense, especially with the following sentence linking to info for _enabling_ modules. And that last sentence seems obsolete now with your addition of step 2.

Reply:
Yeah I thought it was odd, but on one hand you might want to expect the modules are first _disabled_ in order to then justify enabling them, but the fact that they are disabled would be the case 99% of the time this is added. The only time they wouldn't be is if the `app/etc/config.php` already had those modules as enabled (which would mean you can skip step 2 entirely)

I'm happy to change it and remove the last line (linking to how to do it) in place of introducing the step 2 as I have got currently. I think the semantics at this point would assume someone knows the `add a module and enable it` flow in M2.

The other option is we remove the new step 2 and just have some sort of "just so you know here are you modules you should expect to see" to list the modules, then we can keep the internal link (which helps with SEO if nothing else)